terraform/eval_diff.go

```diff
@@ -238,6 +238,38 @@ func (n *EvalDiffTainted) Eval(ctx EvalContext) (interface{}, error) {
 	return nil, nil
 }
 
+// EvalFilterDiff is an EvalNode implementation that filters the diff
+// according to some filter.
```

Comment:
Alright that's fair. I'll make that change (to make it more specific vs. adding a filter callback, just to keep things simpler)